lib/logstash/outputs/elasticsearch/protocol.rb

```diff
@@ -86,11 +94,8 @@ def bulk(actions)
             next { action => args }
           end
         end.flatten)
-        if bulk_response["errors"]
-          return {"errors" => true, "statuses" => bulk_response["items"].map { |i| i["status"] }}
-        else
-          return {"errors" => false}
-        end
+
+        self.class.simple_bulk_response(bulk_response)
```

Comment:
this gets complicated for testing the `HTTPClient` because it depends on `:client_settings` options being passed in, and that is only passed in during an integration with actual connections. I would rather prefer to leave this be a unit test